.github/workflows/upload-docs.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and upload API docs
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to publish docs under (e.g. `v1.2.3-dev.1`)'
+        required: true
+
+env:
+  GH_TOKEN: ${{ github.token }}
```

Comment:
I can't remember, but I think this token has to be generated per repo? 🤔

Reply:
what is the token used for?

Reply:
If I remeber correctly you have to specify it for checkout I think. If the trigger was created by another action as then the default github token won't be valid.

So in our case whatever workflow you have to publish a release will run with the "default" token and will trigger this action, which then has to specify the token explicitly

Reply:
then all our other shared workflows wouldn't work 🤔 I think this fine to remove

Reply:
I removed it. If it doesn't work it is your fault and we have to add it back after the first release :trollface: